R/general_meanplot.R

```diff
@@ -92,9 +98,15 @@ general_meanplot <- function(data,
     p <- p +
       geom_errorbar(aes(ymin = (Mean - SD), ymax = (Mean + SD), color = id_variable), width = 0.4)
   }
-
-  return(p)
-
+  # add sd
+  if (plot_cv) {
+    p <- p +
+      geom_errorbar(aes(ymin = (Mean - CV), ymax = (Mean + CV), color = id_variable), width = 0.4)
+  }
+  # Convert ggplot to plotly
+  p_plotly <- ggplotly(p)
+  
+  return(p_plotly)
```

Comment:
nitpick: [According to the tidyverse guide](https://style.tidyverse.org/functions.html), one should only use `return()` for early returns. I am not a huge fan of this rule, so we can discuss changing it if you prefer - but would need to include those exceptions in our CONTRIBUTING.md file.

```suggestion
   ggplotly(p)
```